Assets/RedBlueGames/MulliganRenamer/Editor/BulkRenamer.cs

```diff
@@ -331,11 +331,7 @@ private static void RenameAsset(UnityEngine.Object asset, string newName)
             if (asset.name != newName)
             {
                 var message = string.Format(
-                                  "Asset [{0}] not renamed when trying to RenameAsset in BulkRenamer. " +
-                                  "It may have been canceled because the new name was already taken by" +
-                                  " an object at the same path. The new name may also have contained " +
-                                  "special characters.\n" +
-                                  "OriginalPath: {1}, New Name: {1}",
+                                  LocaleManager.Instance.GetTranslation("errorAssetNotBulkRenamed"),
```

Comment:
Looks like my original string had a bug with the formatting (uses {1} twice and doesn't use {2} at all). I'm not sure it's possible to see this string right now anyway, since I think I prevent renaming if there will be a problem.

Reply:
Yeah, that was my bad. I forgot to put the string.Format in this one, i will fix it.
If you look at the localization json file and search for "errorAssetNotBulkRenamed", you will see that the string that is there, is exactly the one you wrote, keeping the {}. So we can use string.Format with no problem.

Reply:
Oh, wait... nvm my last message, this translation is inside a string.Format lol.
I will just change the localized text to use the {2} instead of {1}.